lib/ably/realtime/presence.rb

```diff
@@ -265,8 +277,9 @@ def subscribe(*actions, &callback)
       super
     end
 
-    # Unsubscribe the matching block for presence events on the associated Channel.
-    # If a block is not provided, all subscriptions will be unsubscribed
```

Comment:
I think this line is useful and we should keep it

Reply:
Okay, fixed.